src/main/resources/assets/styles/_article.scss

```diff
@@ -56,23 +56,31 @@
 
   .part-fact-box {
     padding: 0;
+    margin: 16px 0;
```

Comment:
Should we use rem here instead? 16px = 1rem

Reply:
Fixed